### ISSUE TITLE
Add const support in array initialization 

### ISSUE THREAD

**Issue by RazrFalcon** [eqrion/cbindgen#53]
Example:

```rust
const LENGTH: usize = 64;

#[repr(C)]
pub struct Data {
    pub d: [u8; LENGTH],
}
```

If I replace LENGTH with a number - everything works just fine.

**Comment by eqrion** [comment]
Fixed in 0.1.27. Please reopen if you are still having an issue with this.

**Comment by RazrFalcon** [comment]
Thanks.
